README.md

```diff
@@ -30,30 +32,46 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+Both of these methods iterate over the items in an array, but '.map' takes things a step further by yielding a new ray, with items that have been copied/manipulated from the original array.
+
 2. What is the difference between a function and a method?
 
+A method is a function that is attached to an object.
+
 3. What is closure?
```

Comment:
you dont need multiple scopes. Just a pair of brackets which make a little environment. those environments or scopes can also be nested